cohttp-eio/tests/test_server.ml

```diff
@@ -1,18 +1,93 @@
 open Cohttp_eio
 
-let read_body req reader =
-  let body = Server.read_fixed req reader in
-  Server.text_response @@ Fmt.str "%a\n\n%s" Http.Request.pp req body
-
-let app (req, reader, _client_addr) =
+let app (req, reader, _) =
   match Http.Request.resource req with
-  | "/get" -> Server.text_response (Fmt.to_to_string Http.Request.pp req)
+  | "/get" ->
+      let buf = Buffer.create 0 in
+      let fmt = Format.formatter_of_buffer buf in
+      Http.Request.pp fmt req;
+      Format.fprintf fmt "%!";
+      Server.text_response (Buffer.contents buf)
```

Comment:
What is this change for? It's much longer and does the same thing.

Reply:
This is need to pretty print the request.

Reply:
The output is identical for me. With `Server.text_response (Fmt.to_to_string Http.Request.pp req)` I get:
```
$ curl http://127.0.0.1:8080/get
meth: GET
resource: /get
version: HTTP/1.1
headers: Header {
 Accept = "*/*"; User-Agent = "curl/7.74.0"; Host = "127.0.0.1:8080" }
```
What do you get with the old version?

Reply:
It seems I misunderstood. Was the code before `Server.text_response (Fmt.to_to_string Http.Request.pp req)`? The latter is shorter, yes. Let me change that.

Reply:
Done.